pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -116,7 +116,7 @@ let
                 then overrideCC stdenv llvmPackages.clangUseLLVM
                 else stdenv;
 
-  nss_pkg = if lib.versionOlder ffversion "83" then nss_3_53 else nss;
+  nss_pkg = if lib.versionOlder ffversion "83" then nss_3_53 else nss.override { useP11kit = false; };
```

Comment:
do you mind adding a comment, for future contributors as well as seems unusual

Reply:
Yes, done.